cmd/protoc-gen-goclay/genhandler/handler.go

```diff
@@ -387,6 +407,10 @@ func (g *Generator) getImplTemplate(f *descriptor.File, s *descriptor.Service, m
 
 	p.Imports = imports
 
+	if useForTest {
```

Comment:
can you please refactor getImplTemplate so it would return `p`? Then `apply*Template` can be called in `generateImplServiceMethod` up the stack.

Reply:
Yep, WIP.

Reply:
Done. Please check it with [41e07dc12847faf2f70d39eae8ff5e022ef7002f](https://github.com/utrack/clay/pull/82/commits/41e07dc12847faf2f70d39eae8ff5e022ef7002f)